spec/controllers/solidus_square/payments_controller_spec.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe 'SolidusSquare::PaymentsController', type: :request do
+  describe "#create" do
+    subject(:endpoint_call) {
+      post '/solidus_square/payments/square', params: params, headers: { "X-Spree-Order-Token": order.guest_token }
+    }
+
+    let(:order) { create(:order) }
+    let(:params) do
+      {
+        payment: {
+          order_number: order.number
+        }
+      }
+    end
+
+    context "with valid params" do
+      before do
+        allow(SolidusSquare::CreatePaymentService).to receive(:call)
```

Comment:
When you stub something don't forget to test the parameters.

Reply:
https://github.com/nebulab/solidus_square/pull/49/commits/b0dd811d394c6ce3eaf59f85907c46318ebb244d